src/sentry/static/sentry/app/components/events/eventCause.tsx

```diff
@@ -86,4 +92,12 @@ class EventCause extends React.Component {
   }
 }
 
+const ExpandButton = styled('button')`
```

Comment:
Please shouldn't we use a Button component here?

Reply:
Maybe, but changing and restyling it seemed outside of typescript conversion scope 👍